docs/optimization_guide/nncf/ptq/basic_quantization_flow.md

```diff
@@ -150,12 +172,24 @@ Tune quantization parameters
        regex = '.*layer_.*'
        nncf.quantize(model, dataset, ignored_scope=nncf.IgnoredScope(patterns=regex))
 
+* ``target_device`` - defines the target device, the specificity of which will be taken into account during optimization. The following values are supported: ``ANY`` (default), ``CPU``, ``CPU_SPR``, ``GPU``, and ``VPU``.
```

Comment:
It would be great to learn more about the effects of setting this parameter. I recently noticed a tiny accuracy degredation on SPR compared to ICL. Will specifying target_device=CPU_SPR fix this? Will that cause lower accuracy on ICL though? (Not asking for an answer, but that's what I wonder when I read this). Will this influence accuracy, performance or both?

Reply:
CPU_SPR option improves model performance on SPR device due to removal of some FQs. It doesn't improve the accuracy, at least it doesn't solve the issue with bf16 enabled by default which I guess is the main reason of the accuracy degradation in your case.